packages/communication-react/review/communication-react.api.md

```diff
@@ -522,7 +522,9 @@ export type ChatCompositeClientState = {
 export type ChatCompositeProps = {
     adapter: ChatAdapter;
     fluentTheme?: PartialTheme | Theme;
-    onRenderAvatar?: (userId: string) => JSX.Element;
+    onRenderAvatar?: (userId: string, avatarType?: 'chatThread' | 'participantList') => JSX.Element;
```

Comment:
this list could get huge (which might be totally fine), but are there other options for what this interface might look like?
e.g.
```
customRenderers?: {
  onRenderAvatar: ()=> JSX.Element,
  onRenderAvatar: ()=> JSX.Element etc.
}
```

(not saying this is better! Just wondering if we explored options for what this customization interface would look like)

Reply:
Then would the documentation highlight what the `onRenderAvatar` would change?

Reply:
Maybe the interface is a good thing to bring up to the azure review board and get their opinions

Reply:
Agree, currently this api allows developer to use just one way to render all the avatar(that could be most likely the case), but if they are willing to, they could render different avatar in different places by second parameter